test/src/test/java/org/corfudb/runtime/object/transactions/AbstractTransactionContextTest.java

```diff
@@ -41,6 +41,10 @@ String put(String key, String value) {
         return getMap().put(key, value);
     }
 
+    String putIfAbsent(String key, String value) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/annymartinez.m/CorfuDB/pullRequest?prid=4651459)